frontend/src/components/widgets/DataFrame/hooks/useTableSizer.ts

```diff
@@ -1,3 +1,5 @@
+// willhuang1997 TODO: Disabling temporarily but need to come back and fix this
```

Comment:
For this file, with the way that it's written, i think it would be a larger refactor to correct the exhaustive dependencies. The reasoning is because whenever something changes, we will execute this:
set size based on containerWidth, then set size based on width passed from proto, then height, then fullscreen, etc. If we add the exhaustive dependencies (size requires height and width), then it would execute setSize based on the fullScreen code which sets the height to the initialWidth.

```// Change sizing if the fullscreen mode is activated or deactivated:
  React.useLayoutEffect(() => {
    if (isFullScreen) {
      const stretchColumns: boolean =
        element.useContainerWidth ||
        (notNullOrUndefined(element.width) && element.width > 0)
      setResizableSize({
        width: stretchColumns ? maxWidth : "100%",
        height: maxHeight,
      })
    } else {
      setResizableSize({
        width: initialWidth || "100%",
        height: initialHeight,
      })
    }
  }, [isFullScreen])
  ```
  
All in all, this would require quite a bit of refactor to get this to deal with the eslint refactor.

Reply:
Yes, that one definitely needs some more attention. The table sizer covers a lot of different situations and configurations. And not everything with this is easy to test in automated settings :(